package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systemic-redis",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
We'll create the releases using the `releae` command, that also generates the changelog. So we don't change the release number manually